server/services/telemetry.py

```
@@ -0,0 +1,135 @@
+
+from __future__ import annotations
+
+import logging
+import threading
+import time
+from collections import deque
+from typing import Deque, Dict, List
+
+logger = logging.getLogger(__name__)
+
+SERIES_LEN = 60  # 60 samples × 1s = 60s window
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)